src/guide/class-and-style.md

```diff
@@ -180,13 +180,13 @@ app.component('my-component', {
 })
 ```
 
-You can learn more about component attribute inheritance in [Non-Prop Attributes](component-attrs.html) section.
+Puede conocer más sobre herencia de atributos de componentes en la sección [Atributos que no son propiedades](component-attrs.html).
```

Comment:
I think we should either remove the uppercase on `Atributos` or change it to:
```suggestion
Puede conocer más sobre herencia de atributos de componentes en la sección [Atributos que no son Propiedades](component-attrs.html).
```

Reply:
I think we can change `Atributos` to `atributos` to keep it consistent with the entire text.

```suggestion
Puede conocer más sobre herencia de atributos de componentes en la sección [atributos que no son propiedades](component-attrs.html).
```